demisto_sdk/commands/common/update_id_set.py

```diff
@@ -1139,6 +1141,35 @@ def get_general_data(path: str, packs: Dict[str, Dict] = None):
     return {id_: data}
 
 
+def get_trigger_data(path: str, packs: Dict[str, Dict] = None):
+    json_data = get_json(path)
+    id_ = json_data.get('RULE_ID', '-')
+    name = json_data.get('name', '')
+    fromversion = json_data.get('fromVersion')
+    toversion = json_data.get('toVersion')
+    pack = get_pack_name(path)
+    marketplaces = get_item_marketplaces(path, item_data=json_data, packs=packs)
+
+    data = create_common_entity_data(path=path, name=name, to_version=toversion, from_version=fromversion, pack=pack, marketplaces=marketplaces)
+
+    return {id_: data}
+
+
+def get_general_yaml_data(path: str, packs: Dict[str, Dict] = None):
+    yaml_data = get_yaml(path)
+
+    id_ = yaml_data.get('id', '-')
```

Comment:
I added a fix, and if there is no `id` so it will be `{pack}-{name}`.
Of course its can be just for now and when things will be more clear we will can remove this.
@dantavori @tkatzir @ShahafBenYakir  WDYT?